test/packages/policy_groups/manifest.yml

```diff
@@ -0,0 +1,111 @@
+description: AWS Integration
+type: integration
+categories:
+  - aws
+  - cloud
+  - network
+  - security
+release: beta
+conditions:
+  kibana.version: '^7.9.0'
+icons:
+  - src: /img/aws-logo.svg
+    title: logo aws
+    size: 32x32
+    type: image/svg+xml
+screenshots:
+  - src: /img/aws-general-dashboard.png
+    title: General AWS dashboard
+    size: 1702x1063
+    type: image/png
+vars:
+  - name: access_key_id
+    type: text
+    title: Access Key ID
+  - name: secret_access_key
+    type: text
+    title: Secret Access Key
+policy_templates:
+  - name: ec2
+    title: AWS EC2
+    description: Collect logs and metrics from EC2 service
+    inputs:
+      - type: s3
+        title: Collect logs from EC2 service
+        description: Collecting EC2 logs using S3 input
+        policy_group: logs
+        vars:
+          - name: visibility_timeout
+            type: text
+            title: Visibility Timeout
+            description: The duration that the received messages are hidden from subsequent retrieve requests after being retrieved by a ReceiveMessage request.  The maximum is 12 hours.
+          - name: api_timeout
+            type: text
+            title: API Timeout
+            description: The maximum duration of AWS API can take. The maximum is half of the visibility timeout value.
+      - type: aws/metrics
+        title: Collect metrics from EC2 service
+        description: Collecting EC2 metrics using AWS CloudWatch
+        policy_group: metrics
+        vars:
+          - name: endpoint
+            type: text
+            title: Endpoint
+            default: "amazonaws.com"
+            description: URL of the entry point for an AWS web service.
+    icons:
+      - src: /img/aws-ec2.svg
+        title: AWS EC2 logo
+        size: 32x32
+        type: image/svg+xml
+    screenshots:
+      - src: /img/aws-ec2-overview.png
+        title: AWS EC2 Overview
+        size: 1702x1063
+        type: image/png
+    vars:
+      - name: ec2_foobar_property
+        type: bool
+        title: EC2 Specific Property
+  - name: dynamodb
+    title: AWS DynamoDB
+    description: Collect logs and metrics from DynamoDB service
+    categories:
+      - datastore
+    inputs:
+      - type: s3
+        title: Collect logs from the DynamoDB service
+        description: Collecting DynamoDB logs using S3 input
+        policy_group: logs
+      - type: aws/metrics
+        title: Collect metrics from DynamoDB service
+        description: Collecting DynamoDB metrics using AWS CloudWatch
+        policy_group: metrics
+    icons:
+      - src: /img/aws-dynamodb.svg
+        title: AWS DynamoDB logo
+        size: 32x32
+        type: image/svg+xml
+    screenshots:
+      - src: /img/aws-dynamodb-overview.png
+        title: AWS DynamoDB Overview
+        size: 1702x1063
+        type: image/png
+    vars:
+      - name: dynamodb_foobar_property
+        type: bool
+        title: DynamoDB Specific Property
+policy_groups:
```

Comment:
Ignoring naming on my end for now ;-)

I'm wondering if the groups should be defined as part of each policy_template or global. Having it local would allow to give more descriptive descriptions. At the same time, I don't think we use it today. Global requires much fewer configuration.

An alternative option is to just not have it in the package at first. Only 2-3 groups are supported, required by the spec and hardcoded into Kibana. Like this everyone is forced to use `logs/metrics` as groups.

Reply:
Actually your both options are related. I agree that global definitions would require less configuration properties and might be easier to setup. The second option is an interesting solution - we can try to kick off the feature with predefined groups and expose only them to users. We need to remember that we won't escape from these tabs in the future :)

Reply:
I'd prefer to start with the most restrictive option we can get away with and then make things less restrictive / more flexible depending on actual usage. So I'm ++ for hardcoding the group names (`logs` or `metrics`) for now. 

How far do we take this hardcoding? I assume not just the group `name`s are hardcoded but also the `title`s — "Logs" or "Metrics". What about the descriptions — should those be somewhat generated, e.g. "Logs for $package_name package"?

Reply:
@sorantis Any thoughts about hardcoding group names (logs, metrics, later on - checks)?

Reply:
responded here https://github.com/elastic/package-spec/pull/137#issuecomment-784100742